wiki/Storyboard/Scripting/Objects/de.md

```diff
@@ -1,70 +1,81 @@
-  - Die Anführungsstriche " sind technisch optional, sie werden jedoch bei Dateinamen und Unterordner benötigt, wenn Leerzeichen darin enthalten sind.
-    - Beispiel: "SB/J\_K.jpg" und nicht SB/J\_K.jpg. Bei SB/J\_K.jpg ist osu! noch in der Lage den Unterordner zu finden, sucht aber ab dann anstatt dem J\_K.jpg ein J.jpg.
-- **(x)** und **(y)** sind **x-/y-Koordinaten an denen das Objekt sein soll.** Die Interpretation hängt vom Wert des (origin) ab; gültige Werte für Instanzen, um ein 640x480 Bild als Hintergrund zu setzen, wären:
-  - origin = TopLeft, x = 0, y = 0
-  - origin = Centre, x = 320, y = 240
-  - origin = BottomRight, x = 640, y = 480
+| Sprite,(Ebene),(Ursprung),"(Dateipfad)",(x),(y) | Animation,(Ebene),(Ursprung),"(Dateipfad)",(x),(y),(frameAnzahl),(frameVerzögerung),(Wiederholung) |
+
+Wobei:
+
+- **(Ebene)** die **[Schicht](/wiki/Storyboard/Scripting/General_Rules) ist, auf denen die Objekte erscheinen.** Die erlaubten Werte werden unten aufgelistet.
+- **(Ursprung)** angibt, wo **auf dem Bild osu! den Ursprung des Bildes (Koordinaten) setzen soll.** Dies beeinflusst die Werte (x) und (y) sowie verschiedene andere befehlsspezifische Verhaltensweisen. Beispielsweise bestimmen die Werte (x) und (y) bei (Ursprung) = TopLeft, wo die linke obere Ecke des Bildes auf dem Bildschirm sein soll. Die erlaubten Werte werden unten aufgelistet.
+- **(Dateipfad)** der **Dateiname des Bildes ist, das du aufrufen möchtest.** Es ist aber nicht immer ganz so einfach:
+  - Wenn du Unterordner im Songordner verwendest, musst du den Pfad miteinbeziehen.
+    - Beispiel: "Hintergründe/himmel.jpg", wenn dein Unterordner "Hintergründe" und das darin beinhaltete Bild "himmel.jpg" heißt. Der Dateipfad fängt erst im Songordner an, also dort wo die .osb- und .osu-Dateien sind (also ein relativer Dateipfad). So etwas wie "C:" ist nicht gültig.
+  - Animationen werden ohne Zahlen referenziert. Wenn du zum Beispiel "sample0.png" und "sample1.png" für eine Animation bestehend aus zwei Frames verwenden möchtest, musst du auf "sample.png" verweisen.
+  - Die Anführungsstriche (`"`) sind technisch optional, sie werden jedoch bei Dateinamen und Unterordnern benötigt, wenn Leerzeichen darin enthalten sind.
+    - Beispiel: "SB/J\_K.jpg" und nicht SB/J\_K.jpg. Bei letzterem sucht osu! nach SB/J (eine ungültige Variable).
+- **(x)** und **(y)** die **x-/y-Koordinaten sind, an denen das Objekt standardmäßig sein soll.** Die Interpretation hängt vom Wert des (Ursprungs) ab. Gültige Werte für Instanzen, um ein 640 x 480 Bild als Hintergrund zu setzen, wären:
+  - Ursprung = TopLeft, x = 0, y = 0
+  - Ursprung = Centre, x = 320, y = 240
+  - Ursprung = BottomRight, x = 640, y = 480
     *und so weiter.*
 
-**Speziell nur für Animationen**
+Ebenen haben diese Werte:
+
+| Wert | Ebene |
+| :-: | :-- |
+| 0 | Background |
```

Comment:
Not so very sure if these should be translated, as they are used in the example image above. If I understand this correctly, you can use both the words as well as the corresponding numerical values for the storyboard commands.